worldedit-core/src/main/java/com/sk89q/worldedit/util/net/HttpRequest.java

```diff
@@ -363,18 +376,20 @@ private static URL reformat(URL existing) {
                     url.getPath(), url.getQuery(), url.getRef());
             url = uri.toURL();
             return url;
-        } catch (MalformedURLException e) {
-            return existing;
-        } catch (URISyntaxException e) {
+        } catch (MalformedURLException | URISyntaxException e) {
             return existing;
         }
     }
 
     /**
-     * Used with {@link #bodyForm(Form)}.
+     * Used with {@link #bodyUrlEncodedForm(Form)}.
      */
     public static final class Form {
-        public final List<String> elements = new ArrayList<>();
+        public final Map<String, String> elements = new LinkedHashMap<>();
+
+        private LazyReference<String> formDataSeparator = LazyReference.from(
```

Comment:
I don't get why we're using `LazyReference` here? Just compute it on initialization, it's not expensive.